### DISCUSSION TITLE
cam timestamp overlay

### DISCUSSION THREAD

**Discussion by lejibxl** [bluenviron/mediamtx#1449] [Questions] [answered]
I use an RPI camera and I would like to be able to add a timestamp overlay to confirm that the camera is still active and not frozen. 
Is this possible?

**Comment by aler9** [comment]
This is a feature in development that is being tracked here: #1288

**Comment by lejibxl** [comment, marked as answer]
Thx aler9 :thumbsup:

[a comment by an automated account is not shown]
